src/cst/parse.js

```diff
@@ -30,5 +38,25 @@ export default function parse(src) {
     return true
   }
   documents.toString = () => documents.join('...\n')
+  if (options.computeLineOffsets) documents.lineOffsets = lf
   return documents
 }
+
+export function charPosToLineCol(offset, lineOffsets) {
```

Comment:
Is there a particular reason why this function is being exported from `cst/parse.js`, rather than its own file?

Reply:
No, just was unsure of your convention on this.